Continue onto all raid sets instead of returning after processing the first. (LP: #401713)
--- a/1.0.0.rc16/lib/metadata/metadata.c
+++ b/1.0.0.rc16/lib/metadata/metadata.c
@@ -839,7 +839,7 @@ _discover_partitions(struct lib_context 
 		 */
 		if (T_GROUP(rs)) {
 			_discover_partitions(lc, &rs->sets);
-			return;
+			continue;
 		}
 
 		/*
